src/components/CountryContent/index.js

```diff
@@ -46,13 +46,12 @@ const useStyles = makeStyles(theme => ({
   selectButtonRoot: {
     width: '100%',
     padding: 0,
-    marginTop: '0.9375rem',
     [theme.breakpoints.up('md')]: {
       marginTop: 0,
       width: '6.609375rem'
     },
     [theme.breakpoints.up('lg')]: {
-      marginLeft: '0.9375rem',
+      marginTop: 0,
```

Comment:
`marginLeft` was used to left align the title and other components
![s](https://user-images.githubusercontent.com/1779590/73726417-91557f80-4740-11ea-9795-d7ea4800c7dd.png)